src/Styleguide/Artwork/Banner.tsx

```diff
@@ -1,22 +1,22 @@
 import React from "react"
-import styled from "styled-components"
 import { Avatar } from "../Elements/Avatar"
-
-const Flex = styled.div`
-  display: flex;
-  flex-direction: row;
-`
+import { Sans, Serif } from "../Elements/Typography"
+import { Flex } from "../Elements/Flex"
 
 export class Banner extends React.Component {
   render() {
     return (
-      <Flex>
+      <Flex flexDirection="row">
         <Avatar size="110px" src="https://picsum.photos/110/110/?random" />
```

Comment:
The `Avatar` size is being set via pixels... should we not?

Reply:
Seems fine for now 👍

Reply:
This seems like a good candidate [for here](https://github.com/artsy/palette/blob/master/src/Theme.tsx#L94), but i would maybe run by brian?

Reply:
I’m thinking that we need to take some more stuff into account in the future (such as fetching the image of the right size, also taking retina into account, etc) but the layout value, to me, seems very local to the component.